### ISSUE TITLE
Ender support

### ISSUE THREAD

**Issue by pier-oliviert** [kbjr/Events.js#4]
Title kinda says it all, I would like to have it included in the NPM :)

Thanks

**Comment by kbjr** [comment]
Hmm.. i have no real experience with ender, but I will start looking into and see what I come up with..

**Comment by pier-oliviert** [comment]
It's pretty trivial to add it, and this is what I use to manage all my loosely coupled module.

**Comment by kbjr** [comment]
Okay, I've looked at this and it looks like ender makes use of commonjs module structures, so something like this should work, correct?

```
if (typeof exports !== 'undefined' && typeof module !== 'undefined' && module.exports) {
  exports = module.exports = Events;
}
```

(again, i've never used ender, so i'm just making sure i understand how it works first.)

**Comment by pier-oliviert** [comment]
Yeah, it seems to be the way to go, You also need to add your package to NPM(Node package manager)

and set your package.json to add dependencies if it has any
